static/js/ducks/followup_requests.js

```diff
@@ -10,9 +10,15 @@ const REFRESH_FOLLOWUP_REQUESTS = "skyportal/REFRESH_FOLLOWUP_REQUESTS";
 
 const PRIORITIZE_FOLLOWUP_REQUESTS = "skyportal/FETCH_FOLLOWUP_REQUESTS";
 
-// eslint-disable-next-line import/prefer-default-export
-export const fetchFollowupRequests = (params = {}) =>
-  API.GET("/api/followup_request", FETCH_FOLLOWUP_REQUESTS, params);
+export function fetchFollowupRequests(params = {}) {
+  if (!Object.keys(params).includes("pageNumber")) {
+    params.pageNumber = 1;
+  }
+  if (!Object.keys(params).includes("numPerPage")) {
+    params.numPerPage = 10;
```

Comment:
Unless you are using different defaults for the front end, I don't see any value in putting the page length and page number as defaults here as well. 

That said, for the number of rows per page, it may be good to have a small default (like 10) for displaying on the front end, and a bigger default for API users.

Reply:
Ok so keep this but make the aPI default larger?

Reply:
I would get rid of the `pageNumber=1`. That's always going to be the default and there's no point having that defined in two places. 
For `numPerPage` you can leave it, with a value that makes sense for the front end page (I guess 10 is fine).

Reply:
good thinking